readme.md

```diff
@@ -0,0 +1,21 @@
+# WordCamp.org
+
+This is a mirror of the [WordCamp directory](https://meta.svn.wordpress.org/sites/trunk/wordcamp.org/) from the WordPress Meta SVN repository, though it does not include externals.
```

Comment:
`SHOULD` - maybe link to the Meta Environment provision for details on getting those? Or maybe #113 will include instructions for Docker instead.